http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -470,6 +473,17 @@ if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
   STEP_TIMINGS+=($(date +%s))
 fi
 
+# RStudio specific setup; only do if RStudio is installed
+if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
+  EXIT_CODE=0
+  retry 3 docker exec ${RSTUDIO_SERVER_NAME} ${RSTUDIO_SCRIPTS}/set_up_package_dir.sh || EXIT_CODE=$?
+  if [ $EXIT_CODE -ne 0 ]; then
+    echo "RStudio user package installation directory creation failed, creating /packages directory"
+    docker exec ${RSTUDIO_SERVER_NAME} /bin/bash -c "mkdir -p ${RSTUDIO_USER_HOME}/packages && chmod a+rwx ${RSTUDIO_USER_HOME}/packages"
+    exit $EXIT_CODE
```

Comment:
We probably don't want to exit here, right?

Reply:
ohh yeah probably not - i added it following the other patterns in the script but this is a different case. I'll remove this but quick question - i tested with an older bioconductor image and the script did not fail. i'm guessing `exit` does not mean the script fails but rather just terminates sooner?

Reply:
Yeah looks like this is near the end anyway (only thing left is `docker prune`) so the image probably started up anyway… still better to not `exit` though. :)